src/coreclr/src/inc/corinfoinstructionset.h

```diff
@@ -293,6 +296,136 @@ inline CORINFO_InstructionSetFlags EnsureInstructionSetFlagsAreValid(CORINFO_Ins
+#endif // TARGET_ARM64
+#ifdef TARGET_AMD64
+        case InstructionSet_SSE :
+            return "SSE";
+        case InstructionSet_SSE_X64 :
+            return "SSE_X64";
+        case InstructionSet_SSE2 :
+            return "SSE2";
+        case InstructionSet_SSE2_X64 :
+            return "SSE2_X64";
+        case InstructionSet_SSE3 :
+            return "SSE3";
+        case InstructionSet_SSSE3 :
+            return "SSSE3";
+        case InstructionSet_SSE41 :
+            return "SSE41";
+        case InstructionSet_SSE41_X64 :
+            return "SSE41_X64";
+        case InstructionSet_SSE42 :
+            return "SSE42";
+        case InstructionSet_SSE42_X64 :
+            return "SSE42_X64";
+        case InstructionSet_AVX :
+            return "AVX";
+        case InstructionSet_AVX2 :
+            return "AVX2";
+        case InstructionSet_AES :
+            return "AES";
+        case InstructionSet_BMI1 :
+            return "BMI1";
+        case InstructionSet_BMI1_X64 :
+            return "BMI1_X64";
+        case InstructionSet_BMI2 :
+            return "BMI2";
+        case InstructionSet_BMI2_X64 :
+            return "BMI2_X64";
+        case InstructionSet_FMA :
+            return "FMA";
+        case InstructionSet_LZCNT :
+            return "LZCNT";
+        case InstructionSet_LZCNT_X64 :
+            return "LZCNT_X64";
+        case InstructionSet_PCLMULQDQ :
+            return "PCLMULQDQ";
+        case InstructionSet_POPCNT :
+            return "POPCNT";
+        case InstructionSet_POPCNT_X64 :
+            return "POPCNT_X64";
+        case InstructionSet_Vector128 :
+            return "Vector128";
+        case InstructionSet_Vector256 :
+            return "Vector256";
+#endif // TARGET_AMD64
+#ifdef TARGET_X86
+        case InstructionSet_SSE :
+            return "SSE";
+        case InstructionSet_SSE2 :
+            return "SSE2";
+        case InstructionSet_SSE3 :
+            return "SSE3";
+        case InstructionSet_SSSE3 :
+            return "SSSE3";
+        case InstructionSet_SSE41 :
+            return "SSE41";
+        case InstructionSet_SSE42 :
+            return "SSE42";
+        case InstructionSet_AVX :
+            return "AVX";
+        case InstructionSet_AVX2 :
+            return "AVX2";
+        case InstructionSet_AES :
+            return "AES";
+        case InstructionSet_BMI1 :
+            return "BMI1";
+        case InstructionSet_BMI2 :
+            return "BMI2";
+        case InstructionSet_FMA :
+            return "FMA";
+        case InstructionSet_LZCNT :
+            return "LZCNT";
+        case InstructionSet_PCLMULQDQ :
+            return "PCLMULQDQ";
+        case InstructionSet_POPCNT :
+            return "POPCNT";
+        case InstructionSet_Vector128 :
+            return "Vector128";
+        case InstructionSet_Vector256 :
+            return "Vector256";
+#endif // TARGET_X86
 
+        default:
+            return "";
```

Comment:
Make this something findable?

```suggestion
            return "UnknownInstructionSet";
```